Sources/BraintreeDropIn/Custom Views/BTVaultedPaymentMethodsTableViewCell.h

```diff
@@ -0,0 +1,21 @@
+#import <UIKit/UIKit.h>
+#import "BTPaymentMethodNonce+DropIn.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol BTVaultedPaymentMethodsTableViewCellDelegate;
+
+@interface BTVaultedPaymentMethodsTableViewCell : UITableViewCell
```

Comment:
This is the table view cell that contains the collection view with the vaulted payment methods.

Reply:
Same note here for adding comment

Reply:
✔️

Reply:
✔️